swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -0,0 +1,263 @@
+				"rt": {
+					"description": "Resource Type",
+					"type": "array",
+					"minItems": 1,
+					"uniqueItems": true,
+					"items": {
+						"maxLength": 64,
+						"type": "string",
+						"enum": [ "oic.r.ael" ]
+					},
+					"readOnly": true
+				},
+				"n": {
+					"$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+				},
+				"id": {
+					"$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+				},
+				"if": {
+					"description": "The OCF Interface set supported by this Resource",
+					"type": "array",
+					"minItems": 2,
+					"uniqueItems": true,
+					"items": {
+						"type": "string",
+						"enum": [ "oic.if.baseline", "oic.if.r" ]
+					},
+					"readOnly": true
+				},
+				"events": {
+					"description": "This list stores AEEs whose 'category' Property value is filtered by 'categoryfilter' Property and 'priority' Property value is equal or less than the value of 'priorityfilter' Property.",
+					"type": "array",
+					"minItems": 1,
+					"uniqueItems": true,
+					"items": {
+						"$ref": "#/definitions/Aee"
+					}
+				},
+				"usedspace": {
+					"description": "Current used space for logged AEEs. The Device updates this Property whenever new AEEs are logged.",
+					"type": "number",
+					"default": 0,
+					"readOnly": true
+				},
+				"maxspace": {
+					"description": "This means the maximum allowable storage size for AEEs that can be stored in 'events' list. The Manufacturer chooses this value.",
+					"type": "number",
+					"readOnly": true
+				},
+				"unit":  {
+					"description": "The unit for 'usedspace' and 'maxspace' Properties. The Manufacturer chooses this value.",
+					"type": "string",
+					"enum": [ 
+						"Kbyte", 
+						"Byte" 
+					],
+					"default": "Kbyte",
+					"readOnly": true
+				},
+				"categoryfilter": {
+					"description" : "This value decides what categories of AEEs are to be logged. Meaning of each bit: 0x01 (Access Control), 0x02 (Onboarding), 0x04 (Device), 0x08 (Authentication), 0x10 (SVR Modification), 0x20 (Cloud), 0x40 (Communication). e.g.) if categoryfilter == 0xff: log all events of all categories,  e.g.) if categoryfilter == 0x03: log all events of 'AC (== 0x01)' and 'OB (==0x02)' categories ",
+					"type": "integer", 
+					"default": 256
+				},
+				"priorityfilter": {
+					"description": "The AEEs whose 'priority' values are equal to or smaller than this value are logged. A smaller value means a higher priority. Meaning of each value: 0 (CRIT), 1 (ERR), 2 (WARN), 3 (INFO), 4 (DEBUG). e.g.) if priorityfilter is set to DEBUG (==4) all AEEs will be logged, e.g.) if priorityfilter is set to 1, CRIT (==0) and ERR (==1) AEEs will be logged ",
+					"type": "integer",
+					"default": 4,
+					"enum": [
+						0, 1, 2, 3, 4
+					]
+				}
+			},
+			"required": [
+				"events", "usedspace", "maxspace"
+			]
+		}
+	}
+}
```

Comment:
It looks like there is an error here...

Reply:
} on line 261 closes "Ael" (line 161)
} on line 262 closes "definitions" (line 96)
} on line 263 closes line 1

Reply:
I added my comment due to the red circle with the arrow after it. When I hover my mouse over it it says: "No newline at end of file"  I'm not sure what this means but it seems like a warning from the internal schema validator.

Reply:
Fixed in https://github.com/openconnectivityfoundation/security/pull/120